recipes/libboxes/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, export_conandata_patches, get, rmdir
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Version
+import os
+
+class libboxesRecipe(ConanFile):
+    name = "libboxes"
+    description = "Boxes is a set of frequently used containers built on top of STL"
+
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://boxes-twdev-projects-ba06c75b02a9b24f40d3a6b9b7130a3266c6a3c532.gitlab.io/"
```

Comment:
```suggestion
    homepage = "https://gitlab.com/twdev_projects/boxes"
```
I think the GitLab repo serves as a slightly better homepage. The Doxygen docs are easy to find from there.

Reply:
Changed as suggested.